sw/applications/example_cpp/main.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 ETH Zurich
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Robert Balas <balasr@iis.ee.ethz.ch>
```

Comment:
Davide Schiavone, Juan Sapriza as authors pls @JuanSapriza

Reply:
I haven't touched these one and the above, should I change them?

Reply:
yes this is for @JuanSapriza